cfme/fixtures/artifactor_plugin.py

```diff
@@ -227,6 +227,7 @@ def pytest_runtest_protocol(item):
     yield
 
 
+@pytest.mark.trylast
 def pytest_runtest_teardown(item, nextitem):
     name, location = get_test_idents(item)
     app = find_appliance(item)
```

Comment:
Yeah I see in L235 we fire the hook `fire_art_test_hook('finish_test')` and if I understand correctly this all happens too early i.e. before the fixture's post-yield block executes. I understand with the decorator that you suggest this problem will be resolved as post-yield will be executed before. 👍